src/skyss0fly/WelcomeMessage/Main.php

```diff
@@ -49,9 +58,10 @@ public function onPlayerLeave(PlayerQuitEvent $event): void {
         if ($leaveMessageEnabled === true) {
             $rawMessage = $config->get("LeaveMessage");
             $message = str_replace("{player}", $player->getName(), $rawMessage);
-            $this->getServer()->broadcastMessage($message);
+            $newMessage = str_replace("{server}", $this->getServer()->getName(), $message);
+            $this->getServer()->broadcastMessage($newMessage);
         } elseif ($leaveMessageEnabled === false) {
-            // No action required
+            return;
```

Comment:
This return does not look good here
I would suggest to check if  `$leaveMessageEnabled` is false and do the return before the if true condition.

Something like this:

```suggestion
       if (!$leaveMessageEnabled) { return; }
       
       if ($leaveMessageEnabled)  {
       // your code
       }
```